test/common/modules_loader.js

```diff
@@ -93,7 +93,7 @@ var modulesLoader = new function() {
 							new Account(scope.db, scope.schema, scope.logger, cb);
 						},
 					},
-					(err, result) => {
+					(__err, result) => {
```

Comment:
why underscore here ?

Reply:
@diego-G btw didn't you want to remove modules_loader once?

Reply:
@yatki When you pass a variable to the callback that you don't use, eslint recommends you to write it that way.

@MaciejBaj I did and I do. I'll open an issue for it after @yatki presents the study about unit.